web-ui/src/main/resources/catalog/components/common/map/mapService.js

```diff
@@ -505,13 +505,15 @@
               source = new ol.source.ImageWMS({
                 params: layerParams,
                 url: options.url,
+                crossOrigin: 'anonymous',
```

Comment:
Seems to break a lot of `getMap` requests.
For me it's not robust and has to be removed.

Reply:
Yep works well with GeoServers and allows to quickly get a PNG image from the map viewer. I'll add an option to turn it off with the export map as PNG actions. https://github.com/geonetwork/core-geonetwork/issues/2149